src/modules/staffAssignment/components/StaffAssignmentCard.tsx

```diff
@@ -49,6 +51,14 @@ const StaffAssignmentCard: React.FC<StaffAssignmentCardProps> = ({
     return Object.entries(staff).sort(([a], [b]) => a.localeCompare(b));
   }, [household]);
 
+  const headOfHousehold = useMemo(() => {
+    if (!household) return;
+
+    return household.householdClients.find(
```

Comment:
We could use the `findHohOrRep` helper here. (But it would have slightly different behavior because it would use the first hh member, not the one whose enrollment dashboard we're on right now)